sql/telemetry_derived/clients_histogram_aggregates_v1/init.sql

```diff
@@ -17,11 +18,12 @@ CREATE TABLE IF NOT EXISTS
       key STRING,
       process STRING,
       agg_type STRING,
-      aggregates ARRAY<STRUCT<key STRING, value INT64>>
```

Comment:
I think the `aggregates` field should be left alone, if it can be used in a similar way as before.